src/database/migrations/2024_02_04_083757_create_place_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('place', function (Blueprint $table) {
+            $table->uuid("id")->primary();
+            $table->string('name');
+            $table->double('lat', 10, 6);
+            $table->double('lon', 10, 6);
+            $table->text('description');
+            $table->timestamps();
+            $table->softDeletes();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::dropIfExists('place');
```

Comment:
Same here : DB tables should be in lower case, with underscores to separate words (snake_case), and should be in plural form.